File: Lab3/Helmet.java
package ua.lviv.iot.hockeyclub;

public class Helmet extends  Good {
	private String sizeOfHead;

	public Helmet () {
		
	}
	
	public Helmet(GoodType goodType, GoodName name, int price , String material, double weight, String manfucturer, String color , int amount,String sizeOfHead) {
		
		super(goodType, name, price, material, weight, manfucturer, color, amount);
		
		this.sizeOfHead = sizeOfHead;
	
	 }
	
	public String getSizeOfHead() {
		return sizeOfHead;
	}

	public void setSizeOfHead(String sizeOfHead) {
		this.sizeOfHead = sizeOfHead;
	}

}

File: Python lab2/goods/Helmet.py
from goods.Good import *
from enums.GoodType import *
from enums.GoodName import *


class Helmet(Good):
    goods_type = GoodType.FORANY
    goods_name = GoodName.SKATES

    def __init__(self, sizeOfHead, name, price, material, weight, manufacturer, color, amount):
        self.sizeofhead = sizeOfHead
        Good.__init__(self, name, price, material, weight, manufacturer, color, amount)

    def __str__(self):
        return "Name : " + str(self.name) + " price: " + str(self.price) + " the size of head is " + str(self.sizeofhead)